### PR TITLE
fix(ci): make upload artifacts job edit the comment

### PR DESCRIPTION
**Pull request by dorimedini-starkware**

### CODE REVIEW

File: .github/workflows/upload_artifacts_workflow.yml [hunk 1]
```diff
@@ -30,20 +30,39 @@ jobs:
...
-          script: |
-            const workflowLink = process.env.WORKFLOW_LINK;
-            github.rest.issues.createComment({
-              issue_number: process.env.PR_NUMBER,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: `Artifacts upload triggered. [View details here](${workflowLink})`
-            })
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ process.env.PR_NUMBER }}
```
[the remark this answers is not in the document]
Comment (dorimedini-starkware) [reply .github/workflows/upload_artifacts_workflow.yml]: Done.

**Review by dorimedini-starkware — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2054)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @graphite-app[bot])

File: crates/blockifier/src/test_utils/struct_impls.rs [hunk 2]
```diff
@@ -213,6 +213,7 @@ impl BlockContext {
     }
 }
 
+// tmp
```
[the remark this answers is not in the document]
Comment (dorimedini-starkware) [reply crates/blockifier/src/test_utils/struct_impls.rs:220]: Done.

**Review by dorimedini-starkware — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2054)* status: 0 of 3 files reviewed, 2 unresolved discussions (waiting on @graphite-app[bot])

**Review by dorimedini-starkware — COMMENTED** [review]
> Dismissed @graphite-app[bot] from 2 discussions.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2054)* status: 0 of 3 files reviewed, all discussions resolved (waiting on @dorimedini-starkware)

**Review by adi-yakovian-starkware — COMMENTED** [review]
> Reviewed 2 of 4 files at r8, 2 of 2 files at r9, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2054)* status: all files reviewed, 1 unresolved discussion (waiting on @dorimedini-starkware)

___
*[`.github/workflows/upload_artifacts_workflow.yml` line 38 at r9](https://reviewable.io/reviews/starkware-libs/sequencer/2054#-OCKQf9j1xN-RFBAIRGc:-OCKQf9k4Trbrd5FfFUi:b-1x9l6c) ([raw file](https://github.com/starkware-libs/sequencer/blob/94928f3bccddcc036a35d5f60437471c2956d725/.github/workflows/upload_artifacts_workflow.yml#L38)):*
> ```YAML
>         if: env.PR_NUMBER != ''
>         uses: starkware-libs/find-comment@v3
>         id: fc
> ```

I prefer to avoid abbreviations, unless you have to.
It's your call, though (non-blocking), I don't know the conventions in this repo.



_Suggestion:_
```YAML
find-comment
```

**Review by adi-yakovian-starkware — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2054#-:-OCSODeB1UujogOuJe--:bnfp4nl)* status: all files reviewed, 3 unresolved discussions (waiting on @dorimedini-starkware)

___
*[`.github/workflows/upload_artifacts_workflow.yml` line 46 at r9](https://reviewable.io/reviews/starkware-libs/sequencer/2054#-OCSM_NaE5Xhi94qiD-3:-OCSM_Nb0bkvhGsbXqAd:b-8lfetr) ([raw file](https://github.com/starkware-libs/sequencer/blob/94928f3bccddcc036a35d5f60437471c2956d725/.github/workflows/upload_artifacts_workflow.yml#L46)):*
> ```YAML
> 
>       - name: Create comment
>         if: env.PR_NUMBER != '' && steps.fc.outputs.comment-id == ''
> ```

Please document, can be done in another PR if you prefer.



_Code quote:_
```YAML
if: env.PR_NUMBER != '' && steps.fc.outputs.comment-id == ''
```
___
*[`.github/workflows/upload_artifacts_workflow.yml` line 56 at r9](https://reviewable.io/reviews/starkware-libs/sequencer/2054#-OCSMo-s4uB0837H8Eip:-OCSMo-s4uB0837H8Eiq:brhumzt) ([raw file](https://github.com/starkware-libs/sequencer/blob/94928f3bccddcc036a35d5f60437471c2956d725/.github/workflows/upload_artifacts_workflow.yml#L56)):*
> ```YAML
> 
>       - name: Update comment
>         if: env.PR_NUMBER != '' && steps.fc.outputs.comment-id != ''
> ```

Same here.



_Code quote:_
```YAML
if: env.PR_NUMBER != '' && steps.fc.outputs.comment-id != ''
```

**Review by adi-yakovian-starkware — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r10, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2054#-:-OCSi3re5R29kd8sQtLx:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @dorimedini-starkware)

**Review by adi-yakovian-starkware — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/2054#-:-OCSxOeH2ZqC2PoFfIv6:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @dorimedini-starkware)

**Pull request merged**